mypy.ini

```
@@ -13,3 +13,51 @@ explicit_package_bases = True
 disable_error_code =
 	# Disable due to many false positives
 	overload-overlap,
+	# TODO: Consider Annotated and/or replacing autocommand jaraco/jaraco.develop#20
+	valid-type,
+	syntax,
```

Thanks for consolidating this concern. I'm a little worried that this exclusion could linger after autocommand is replaced, but since it explicitly mentions autocommand, I guess that's enough to either notice it when removing autocommand or at least determine that it's no longer needed after autocommand is gone.

I don't think it's possible to tell that the comment applies to both lines. Perhaps there should be a blank line above the TODO, suggesting that this comment and the two lines that follow it are a separate section of `disable_error_code`.
```suggestion

	# TODO: Consider Annotated and/or replacing autocommand jaraco/jaraco.develop#20
	valid-type,
	syntax,
```